### ISSUE TITLE
Teams Update

### ISSUE THREAD

**Issue by mcainiom** [microsoft/shell-intune-samples#13]
We are having issues with Teams being out of date and no longer working without an update. The script aborts if the Application is already present, can the autoupdate variable be overridden so the modified date is checked and a version is installed anyway if the meta file is not present?

**Comment by theneiljohnson** [comment]
Yes, set autoUpdate="false" and it will overwrite the current install.

**Comment by mcainiom** [comment]
Thanks, ended up forking the script and making modifications. Couldn't work out how to pass properties/variables using the quick run command in Intune

**Comment by theneiljohnson** [comment]
We can’t pass variables to scripts yet. It’s on our roadmap, hopefully before the end of the year.

Feel free to let me have any recommendations, I’d love to see what you had to change.

From: mcainiom ***@***.***>
Date: Monday, 19 April 2021 at 16:20
To: microsoft/shell-intune-samples ***@***.***>
Cc: Neil Johnson ***@***.***>, Comment ***@***.***>
Subject: Re: [microsoft/shell-intune-samples] Teams Update (#13)

Thanks, ended up forking the script and making modifications. Couldn't work out how to pass properties/variables using the quick run command in Intune

**Comment by mcainiom** [comment]
It was only the autoupdate I changed. I also added a check so when an internal server returned a 200 it would download from our internal web server to save the pipe, a simplified version of what you've done on the office install script. A small cosmetic change I made was adding the ':' to the last-modified date just to cleanup the guff that's returned.

Some really nice, useful scripts though, definitely changed my attitude/mindset towards packaging software. Where possible I just script the download from the source and install and don't touch the package. Office I have a script to write out our install customisation xml and let the downloaded package pick that up. Life changing :D
